tests/amazon/amazon-music.js

```diff
@@ -0,0 +1,61 @@
+module.exports = {
+    'Verify user can navigate to AmazonUS by inputting zip code': function (browser) {
+        const amazonHome = browser.page.amazonHomePage();
+
+        amazonHome.navigate();
+        amazonHome.waitForElementVisible('@locationButton', 15000);
+        amazonHome.click('@locationButton');
+        amazonHome.waitForElementVisible('@zipInput', 15000);
+        amazonHome.setValue('@zipInput', '53186');
+        amazonHome.click('@zipSubmitButton');
+        browser.pause(1000); // mandatory pause to allow one second threshhold for US confirmation prompt
+
+        browser.element('css selector', '[class="a-size-small,a-text,a-color-tertiary,a-spacing-top-base"]', function(result){
+            if (result.value && result.value.ELEMENT) {
+                amazonHome.waitForElementVisible('@finishButton');
+                amazonHome.click('@finishButton');
+            } else {
+                amazonHome.waitForElementVisible('@doneButton');
+                amazonHome.click('@doneButton');
+            }
+        });
+
+        amazonHome.waitForElementVisible('@mainAppFrame', 15000);
+    },
+
+    'Verify user can go to music library by Nav list': function (browser) {
+        const amazonHome = browser.page.amazonHomePage();
+        const amazonMusic = browser.page.amazonMusicPage();
+
+        browser.pause(3000); //mandatory pause to wait for app to consistently land on English
+        amazonHome.moveToElement('@accountButton', 1, 1);
+        // amazonHome.moveToElement('@accountButton', 1, 1); //make sure that it tiggers the list
```

Comment:
duplicated with line 31, is this expected? please investigate and remove if should